test/opencensus/trace_test.exs

```diff
@@ -0,0 +1,90 @@
+defmodule Opencensus.TraceTest do
+  use ExUnit.Case
+  use ExUnitProperties
+
+  import Opencensus.Trace
+
+  setup do
+    :application.load(:opencensus)
+    :application.set_env(:opencensus, :send_interval_ms, 1)
+    :application.set_env(:opencensus, :reporter, {PidAttributesReporter, self()})
```

Comment:
This needs to be changed before `0.8.0` of OpenCensus as this have changed to `:reporters` and list.

Reply:
Yeah, we missed _that_ bus.